documentation/tut/forecasting_scheduling.rst

```diff
@@ -24,16 +24,16 @@ Start to run one worker for each kind of job (in a separate terminal):
 
 .. code-block:: console
 
-   flexmeasures run-worker --queue forecasting
-   flexmeasures run-worker --queue scheduling
+   flexmeasures jobs run-worker --queue forecasting
+   flexmeasures jobs run-worker --queue scheduling
```

Comment:
I get `Error: Missing option '--name'.` Maybe we can automatically assign a name?

Related issue: https://github.com/FlexMeasures/flexmeasures/issues/452